spec/controllers/placeholder_users_controller_spec.rb

```diff
@@ -129,155 +87,226 @@
       end
     end
 
-    before do
-      as_logged_in_user(current_user) do
-        get :edit, params: { id: placeholder_user.id }
+    describe 'POST create' do
+      let(:params) do
+        {
+          placeholder_user: {
+            name: 'UX Developer'
+          }
+        }
       end
-    end
 
-    context 'as admin' do
-      it_behaves_like 'renders the edit template'
-    end
+      before do
+        post :create, params: params
+      end
 
-    context 'not as admin' do
-      let(:current_user) { FactoryBot.build(:user) }
+      context 'without ee' do
+        it 'returns with an error' do
+          expect { post :create, params: params }.not_to change { PlaceholderUser.count }
+          expect(response).to be_successful
```

Comment:
Looks counterintuitive to me that a failed resource creation returns a success status but errors. "Hey, great, all good, we found errors." hmmm. But I guess, that this is how we do it, right?

Reply:
Yes I think it's common to render an existing template which then results in a 200 OK response even though there are invalid fields. I think that this is okay, given that this is not an API response.